gadget/gadget.go

```diff
@@ -328,7 +330,7 @@ func ReadInfo(gadgetSnapRootDir string, classic bool) (*Info, error) {
 		switch v.Bootloader {
 		case "":
 			// pass
-		case "grub", "u-boot", "android-boot":
+		case "grub", "u-boot", "android-boot", "lk":
```

Comment:
do we need a test for this change, we don't have one for android-boot but not sure it's a reason

Reply:
Test added for this